src/EditorFeatures/Test2/Rename/RenameEngineTests.VisualBasicConflicts.vb

```diff
@@ -1181,14 +1181,14 @@ End Module
                                     Protected Class [|$$A|]
                                     End Class
                                 End Class
-                                
+
```

Comment:
What is removing whitespace in XML doc comments? Are you doing it manually or is it via some tool? If being done by a tool, this could be bad in some cases (not this PR specifically though).